CFC_WebApp/main/tripManager.py

```diff
@@ -87,6 +41,13 @@ def stripoutNonSerializable(sectionList):
         strippedList.append(section)
     return strippedList
 
+def filter_unclassifiedSections(UnclassifiedSections):
+    filtered_Sections=[]
+    for section in UnclassifiedSections:
+        if section['filtered']:
```

Comment:
1,2,3. Yeah there is a difference between the spec and the code. I will change the spec and then maybe change the new field name. I'm not sure what an appropriate name would be though, other than `keep` or something along those lines.

4. That is an interesting point, but the tests do pass. I will double check that though.